docs/examples/infer-orientation-daily-peak.py

```diff
@@ -0,0 +1,87 @@
+"""
+Infer Array Tilt/Azimuth - Daily Peak
+=====================================
```

Comment:
Making a note here to come back and decide what to do with this example after the discussion next week.